docs/faq/add_op_in_backend.md

```diff
@@ -601,8 +646,8 @@ using nnvm. Congratulations! You now know how to add operators.
 We welcome your contributions to MXNet.
 
 **Note**: Source code in the tutorial can be found in
-[quadratic_op-inl.h](https://github.com/reminisce/mxnet/blob/add_op_example_for_tutorial/src/operator/tensor/quadratic_op-inl.h),
-[quadratic_op.cc](https://github.com/reminisce/mxnet/blob/add_op_example_for_tutorial/src/operator/tensor/quadratic_op.cc),
-[quadratic_op.cu](https://github.com/reminisce/mxnet/blob/add_op_example_for_tutorial/src/operator/tensor/quadratic_op.cu),
+[quadratic_op-inl.h](https://github.com/apache/incubator-mxnet/blob/master/src/operator/contrib/quadratic_op-inl.h),
```

Comment:
The file/line number on master branch may change in the future. Suggest to refer to the same file in v1.3.x branch.

Reply:
but won't it be wrong that we will point users to v1.3.x branch and this change won't even be in that branch the v1.3 cut has already been made and this PR is not part of it.

Reply:
alright I didnt' notice you also made code changes.